app/src/main/java/com/mwalasiak/pyramid/LevelsActivity.kt

```diff
@@ -8,8 +8,6 @@ import android.os.Bundle
 import android.widget.EditText
 import android.widget.TextView
 import android.widget.Toast
-import com.mwalasiak.pyramid.api.LottoApi
-import com.mwalasiak.pyramid.api.OkHttpLoggingInterceptor
 import com.mwalasiak.pyramid.databinding.ActivityLevelsViewBinding
 
 
```

Comment:
Remember about formatting the lines in the file after all the changes ;) 
<img width="266" alt="Screenshot 2022-08-11 at 10 08 42" src="https://user-images.githubusercontent.com/19372220/184091019-bd64aabb-8305-496c-a38f-3fe1858c76a4.png">